index.js

```diff
@@ -10,142 +10,125 @@ server.get("/", (req, res) => {
+    res.status(200).json(edits);
+  } catch (error) {
+    console.log(error);
+    res
+      .status(500)
+      .json({ error: "This action can't be updated at this time." });
+  }
+});
+
+server.put("/api/projects/:id", (req, res) => {
   dbp
-    .insert(req.body)
-    .then(result => {
-      res.status(200).json(result);
+    .update(req.params.id, req.body)
+    .then(response => {
+      res.status(200).json(response);
     })
     .catch(error => {
-      res.status(500).json({ error: "The project(s) couldn't be added." });
+      res
+        .status(500)
+        .json({ error: "This project can't be updated at this time." });
     });
 });
 
-
-server.put('/api/actions/:id', (req, res) => {
-    dba.update(req.params.id, req.body)
-        .then(response => {
-           res
-                .status(200)
-                .json(response)
-        })
-        .catch(error => {
-            res
-                .status(500)
-                .json({error: "This action can't be updated at this time." })
-        })
-});
-
-
-server.put('/api/projects/:id', (req, res) => {
-    dbp.update(req.params.id, req.body)
-        .then(response => {
-            res
-                .status(200)
-                .json(response)
-        })
-        .catch(error => {
-            res
-                .status(500)
-                .json({error: "This project can't be updated at this time." })
-        })
-})
-
-
-server.delete('/api/actions/:id', (req,res) =>{
-    dba.remove(req.params.id,)
+server.delete("/api/actions/:id", (req, res) => {
+  dba
+    .remove(req.params.id)
     .then(response => {
-        res.status(200).json(response)
-    })
-    .catch(error =>{
-        res.status(400).json({message:"Action not deleted"})
+      res.status(200).json(response);
     })
-})
-
+    .catch(error => {
+      res.status(400).json({ message: "Action not deleted" });
+    });
+});
 
-server.delete('/api/projects/:id', (req,res) =>{
-    dbp.remove(req.params.id,)
+server.delete("/api/projects/:id", (req, res) => {
+  dbp
+    .remove(req.params.id)
     .then(response => {
-        res.status(200).json(response)
-    })
-    .catch(error =>{
-        res.status(400).json({message:"Project not deleted"})
+      res.status(200).json(response);
     })
-})
-
+    .catch(error => {
+      res.status(400).json({ message: "Project not deleted" });
+    });
+});
 
-server.get("/api/projects/:id/actions", (req, res) => {
-    const id = req.params.id;
-    dbp
-      .getProjectActions(id)
-      .then(posts => {
-        res.status(200).json(posts);
-      })
-      .catch(err => {
-        res.status(400).json({ message: "Not found" });
-      });
-  });
+server.get("/api/projects/:id/actions", async (req, res) => {
+  try {
+    const id = await dbp.getProjectActions(req.params.id);
+    res.status(200).json(id);
+  } catch (error) {
+    console.log(error);
+    res.status(400).json({ message: "Not found" });
+  }
+});
```

Comment:
All endpoints working, server methods are written well, there is a sanity check endpoint present. :heavy_check_mark: 

I would like to see the routes broken up, though :thinking: